cmd/tracee-ebpf/flags/capabilities.go

```diff
@@ -0,0 +1,105 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aquasecurity/tracee/pkg/capabilities"
+	"github.com/aquasecurity/tracee/pkg/filters"
+	"kernel.org/pub/linux/libs/security/libcap/cap"
+)
+
+const (
+	CapsMainFlag        = "caps"
+	AllowFailedDropFlag = "allow-failed-drop"
+	CancelDropFlag      = "cancel-drop"
+	AddReqCapsFlag      = "add"
+)
+
+func CapabilitiesHelp() string {
+	return fmt.Sprintf(`Manage the capabilities and capabilities-related operations of tracee-ebpf
+Normally, tracee will drop all capabilities not required to its operations.
+
+Possible options:
+  %-51s allow tracee-ebpf to run with high capabilities, in case that capabilities dropping fails
+  %-51s cancel capabilities drop, so tracee will run with all given capabilities (unless missing required capabilities)
+  %-51s add required capabilities, so tracee won't drop them
+`,
+		AllowFailedDropFlag,
+		CancelDropFlag,
+		fmt.Sprintf("%s=<list_of_caps>", AddReqCapsFlag))
+}
+
+type CapsConfig struct {
+	AllowHighCaps  bool
+	CancelCapsDrop bool
+	CapsToPreserve []cap.Value
+}
+
+func PrepareCapsConfig(capsCfgArray []string) (CapsConfig, error) {
+	var cfg CapsConfig
+	for _, opt := range capsCfgArray {
+		optName := opt
+		var operatorAndValues string
+		operatorIndex := strings.IndexAny(opt, "=!<>")
+		if operatorIndex > 0 {
+			optName = opt[0:operatorIndex]
+			operatorAndValues = opt[operatorIndex:]
+		}
+
+		switch optName {
+		case AllowFailedDropFlag:
+			cfg.AllowHighCaps = true
+		case CancelDropFlag:
+			cfg.CancelCapsDrop = true
+		case AddReqCapsFlag:
+			addedCapsFilter := filters.StringFilter{}
+			err := addedCapsFilter.Parse(operatorAndValues)
+			if err != nil {
+				return cfg, err
+			}
+			if addedCapsFilter.FilterOut() {
+				capsFilteredOut, err := capsStringSliceToValues(addedCapsFilter.NotEqual)
+				if err != nil {
+					return cfg, err
+				}
+				rcaps := capabilities.GetAllCapabilities()
+				for _, c := range capsFilteredOut {
+					rcaps = removeCapFromSlice(rcaps, c)
+				}
+				cfg.CapsToPreserve = rcaps
+			} else {
+				cfg.CapsToPreserve, err = capsStringSliceToValues(addedCapsFilter.Equal)
+				if err != nil {
+					return cfg, err
+				}
+			}
+		default:
+			return cfg, fmt.Errorf("illegal flag '--%s %s'. Run '--%[1]s help' for more info", CapsMainFlag, optName)
+		}
+	}
+	return cfg, nil
+}
+
+func capsStringSliceToValues(slice []string) ([]cap.Value, error) {
+	var caps []cap.Value
+	if len(slice) > 0 {
+		for _, cName := range slice {
+			c, err := cap.FromName(cName)
```

Comment:
Ok, just checked this and it works but only for lower case capability names. 
So `cap_sys_admin` works but `CAP_SYS_ADMIN` doesn't, I assume library uses only lower case names for some reason. 
But in the [man page](https://man7.org/linux/man-pages/man7/capabilities.7.html) capabilities are upper case so they should probably also be supported.